draw/draw_recon_viewer.py

```diff
@@ -53,6 +54,17 @@ def test_render_2():
 
 test_render_2()
 
+def test_render_3():
+    host_leaves = parasite_leaves = 12
+    recon_input = input_generator.generate_random_recon_input(host_leaves, parasite_leaves)
+    recongraph = recon_input.reconcile(1, 1, 1)
+    reconciliation = recongraph.median()
+
+    fig = recon_viewer.render(recon_input.host_dict, recon_input.parasite_dict, reconciliation._reconciliation, 
+                                reconciliation.event_frequencies, show_internal_labels=True, show_freq=True)
+    filepath = Path(common.output_path).joinpath("test_render_3.png")
+    fig.save(filepath)
+
```

Comment:
Please call it `test_render_3()` here, so that the output gets saved. :)
```python
test_render_3()
```

Reply:
Oops, fixed!